src/main/java/nl/tudelft/jpacman/Launcher.java

```diff
@@ -177,10 +183,21 @@ public void launch() {
 		addSinglePlayerKeys(builder, game);
 		pacManUI = builder.build(game);
 		pacManUI.start();
+		//IApac();
+
+	}
+
+	/**
+	 * 
+	 */
+	public void IApac() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not start with capital letters](https://www.codacy.com/app/mikis-millemaci/jpacman-framework/file/2525539198/issues/source?bid=3256381&fileBranchId=3281826#l193)